diff --git a/tutorials/3-Practical/train_alpaca.py b/tutorials/3-Practical/train_alpaca.py
index bb9ba55..f3a2093 100644
--- a/tutorials/3-Practical/train_alpaca.py
+++ b/tutorials/3-Practical/train_alpaca.py
@@ -34,6 +34,6 @@ train(start_date = '2022-6-11',
       API_SECRET = API_SECRET, 
       API_BASE_URL = API_BASE_URL,
       erl_params=ERL_PARAMS,
-      cwd='./papertrading_erl', #current_working_dir
+      cwd='./papertrading_erl1', #current_working_dir
       break_step=1e7)
 
diff --git a/tutorials/3-Practical/wandb/latest-run b/tutorials/3-Practical/wandb/latest-run
index 7abd308..0dc1820 120000
--- a/tutorials/3-Practical/wandb/latest-run
+++ b/tutorials/3-Practical/wandb/latest-run
@@ -1 +1 @@
-run-20220913_095854-1fpv0mue
\ No newline at end of file
+run-20220913_163028-1z48is09
\ No newline at end of file
